githubutils/repo_client_test.go

```diff
@@ -100,40 +117,48 @@ var _ = Describe("repo client utils", func() {
 		expectStatus(loaded, status)
 	}
 
-	It("can manage status", func() {
-		client = githubutils.NewRepoClient(githubClient, owner, repo)
-		// randomizing this would create a (slim) potential race
-		// not randomizing makes it less easy to validate that the create worked, but we'll assume github api responses are accurate
-		status := &github.RepoStatus{
-			State:       github.String(githubutils.STATUS_SUCCESS),
-			Context:     github.String("test"),
-			Description: github.String("test"), // longer than 140 characters will be truncated
-		}
-		testManageStatus(client, status, sha)
-	})
+	Context("With write token", func() {
+		BeforeEach(func() {
+			if os.Getenv("HAS_CLOUDBUILD_GITHUB_TOKEN") == "" {
+				Skip("Needs write token")
+			}
+		})
```

Comment:
As with the Gcloud stuff, these need write tokens and you can't safely give out write tokens to random people, so they have to be skipped. This is a problem, but, there isn't really a better approach. If it matters, a member should make their own branch based on the contributor's branch and make a new PR from that.